app/views/accessories/new.html.haml

```diff
@@ -1,30 +1,29 @@
 .modal-header
   = modal_close_button
-  %h3= t('product_accessories.pick_accessories.title')
-= simple_form_for :accessories, :url => url_for(action: :create), :remote => request.xhr?, :html => { 'data-type' => 'html', :id => 'accessory-form', :class => :pick_accessories_form } do |f|
+  %h3= t("product_accessories.pick_accessories.title")
+= simple_form_for :accessories, url: url_for(action: :create), remote: request.xhr?, html: { "data-type" => "html", :id => "accessory-form", class: :pick_accessories_form } do |f|
```

Comment:
This is fine, but fyi for data attributes, you can use `data: { type: "html" }`. If you have an underscore in the keys, it'll get translated to a hyphen in the html.